cdc/scheduler/internal/v3/replication/replication_set.go

```diff
@@ -195,6 +241,75 @@ func NewReplicationSet(
 	return r, nil
 }
 
+func (r *ReplicationSet) hasRole(role CaptureRole) bool {
+	_, has := r.getRole(role)
+	return has
+}
+
+func (r *ReplicationSet) isInRole(captureID model.CaptureID, role CaptureRole) bool {
```

Comment:
just name it `inRole` should be ok.

Reply:
The usual convention to name methods that return boolean is to prefix verbs such as 'is' or 'has' to the predicate as a question, or use the predicate as an assertion.